api/base/views.py

```diff
@@ -6,57 +6,214 @@
 
 @api_view(('GET',))
 def root(request, format=None):
-    """
-        Welcome to the V2 Open Science Framework API. With this API you can programatically access users,
-        projects, components, and files from the [Open Science Framework](https://osf.io/). The Open Science
-        Framework (OSF) is a free, open-source service maintained by the [Center for Open Science](http://cos.io/).
+    """Welcome to the V2 Open Science Framework API. With this API you can access users, projects, components, and files
+    from the [Open Science Framework](https://osf.io/). The Open Science Framework (OSF) is a free, open-source service
+    maintained by the [Center for Open Science](http://cos.io/).
+
+    The OSF serves as a repository and archive for study designs, materials, data, manuscripts, or anything else
+    associated with your research during the research process. Every project and file on the OSF has a permanent unique
+    identifier, and every registration (a permanent, time-stamped version of your projects and files) can be assigned a
+    DOI/ARK. You can use the OSF to measure your impact by monitoring the traffic to projects and files you make
+    public. With the OSF you have full control of what parts of your research are public and what remains private.
+
+    Beta notice: This API is currently a beta service.  You are encouraged to use the API and will receive support
+    when doing so, however, while the API remains in beta status, it may change without notice as a result of
+    product updates. The temporary beta status of the API will remain in place while it matures. In a future
+    release, the beta status will be removed, at which point we will provide details on how long we will support
+    the API V2 and under what circumstances it might change.
+
+    #General API Usage
+
+    The OSF API generally conforms to the [JSON-API v1.0 spec](http://jsonapi.org/format/1.0/).  Where exceptions
+    exists, they will be noted.  Each endpoint will have its own documentation, but there are some general principles.
+
+    ##Requests
+
+    ###Canonical URLs
+
+    All canonical URLs have trailing slashes.  A request to an endpoint without a trailing slash will result in a
+    301 redirect to the canonical URL.
```

Comment:
If we provide a link that does not end in a slash, though, don't add a slash to the end. (This will mostly happen with files.)